md-to-pl.py

```diff
@@ -0,0 +1,109 @@
+import os
+import uuid
+import json
+import md_parser
+
+# TODO: validity checking for customizations
+ALLOWED_CUSTOMIZATIONS = {'multiple-choice': ['answers-name', 'weight', 'inline', 'number-answers', 'fixed-order',
+                                              'hide-letter-keys', 'all-of-the-above', 'none-of-the-above',
+                                              'external-json', 'external-json-correct-key',
+                                              'external-json-incorrect-key'],
+                          'checkbox': ['answers-name', 'weight', 'inline', 'number-answers', 'min-correct',
+                                       'max-correct', 'fixed-order', 'partial-credit', 'partial-credit-method',
+                                       'hide-help-text', 'detailed-help-text', 'hide-answer-panel', 'hide-letter-keys',
+                                       'hide-score-badge'],
+                          'number-input': ['answers-name', 'weight', 'correct-answer', 'label', 'suffix', 'display',
+                                           'comparison', 'rtol', 'atol', 'digits', 'allow-complex', 'allow-blank',
+                                           'blank-value', 'show-help-text', 'show-placeholder', 'size',

```

Comment:
@firasm to open an issue in PL to avoid throwing an `UnknownAttribute` Exception if the wrong parameter is passed in

Reply:
Suggestion : let question fail on PrairieLearn if the wrong attributes are passed, or if the attributes are the wrong type.

Reply:
resolved by removing in #20